internal/fields/model.go

```diff
@@ -9,6 +9,7 @@ type FieldDefinition struct {
 	Name        string            `yaml:"name"`
 	Description string            `yaml:"description"`
 	Type        string            `yaml:"type"`
+	Value       string            `yaml:"value"` // The value to associate with a constant_keyword field.
```

Comment:
As the spec is merged, you need to update the Go dependency on the package-spec.

Reply:
Updated to https://github.com/elastic/package-spec/commit/b358e974b7f9ce78f05970c4880a7a1dbdb35e59.